runners/spark/job-server/build.gradle

```diff
@@ -49,14 +49,13 @@ configurations.all {
 }
 
 dependencies {
-  compile project(sparkRunnerProject)
```

Comment:
@sonam-vend All Spark Python tests are failing. I suspect this line is the cause.